### PR TITLE
MODELIX-574: feat(docker): dockerize use cases

### PR DESCRIPTION
**Pull request by nkoester**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,20 +1,38 @@
 services:
+
   model-server:
     # Keep the version in sync with the modelix version in gradle/libs.versions.toml
-    image: modelix/modelix-model:3.11.0
+    image: modelix/model-server:3.11.0
+    profiles:
+      - uc1
```
Comment (odzhychko) [inline docker-compose.yaml:7]: I would like to name the profiles, `useCase1a`, `useCase1b` and `useCase2` to make the better attributable to the use case numbers in the upcoming sample documentation.

Comment (nkoester) [reply docker-compose.yaml]: done

**Comment by mhuster23** [comment]
```
spa-overview-angular  | 2023/11/10 14:42:27 [error] 11#11: *1 directory index of "/usr/share/nginx/html/" is forbidden, client: 192.168.65.1, server: , request: "GET / HTTP/1.1", host: "localhost:4200"
spa-overview-angular  | 192.168.65.1 - - [10/Nov/2023:14:42:27 +0000] "GET / HTTP/1.1" 403 153 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
spa-overview-angular  | 2023/11/10 14:42:27 [error] 11#11: *1 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 192.168.65.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:4200", referrer: "http://localhost:4200/"
spa-overview-angular  | 192.168.65.1 - - [10/Nov/2023:14:42:27 +0000] "GET /favicon.ico HTTP/1.1" 500 177 "http://localhost:4200/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
spa-management-vue    | 2023/11/10 14:42:32 [error] 9#9: *1 directory index of "/usr/share/nginx/html/" is forbidden, client: 192.168.65.1, server: , request: "GET / HTTP/1.1", host: "localhost:3000"
spa-management-vue    | 192.168.65.1 - - [10/Nov/2023:14:42:32 +0000] "GET / HTTP/1.1" 403 153 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
spa-management-vue    | 2023/11/10 14:42:32 [error] 9#9: *1 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 192.168.65.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:3000", referrer: "http://localhost:3000/"
spa-management-vue    | 192.168.65.1 - - [10/Nov/2023:14:42:32 +0000] "GET /favicon.ico HTTP/1.1" 500 177 "http://localhost:3000/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
```
@odzhychko have you encountered this as well?

**Comment by nkoester** [comment]
> ```
> spa-overview-angular  | 2023/11/10 14:42:27 [error] 11#11: *1 directory index of "/usr/share/nginx/html/" is forbidden, client: 192.168.65.1, server: , request: "GET / HTTP/1.1", host: "localhost:4200"
> spa-overview-angular  | 192.168.65.1 - - [10/Nov/2023:14:42:27 +0000] "GET / HTTP/1.1" 403 153 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
> spa-overview-angular  | 2023/11/10 14:42:27 [error] 11#11: *1 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 192.168.65.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:4200", referrer: "http://localhost:4200/"
> spa-overview-angular  | 192.168.65.1 - - [10/Nov/2023:14:42:27 +0000] "GET /favicon.ico HTTP/1.1" 500 177 "http://localhost:4200/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
> spa-management-vue    | 2023/11/10 14:42:32 [error] 9#9: *1 directory index of "/usr/share/nginx/html/" is forbidden, client: 192.168.65.1, server: , request: "GET / HTTP/1.1", host: "localhost:3000"
> spa-management-vue    | 192.168.65.1 - - [10/Nov/2023:14:42:32 +0000] "GET / HTTP/1.1" 403 153 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
> spa-management-vue    | 2023/11/10 14:42:32 [error] 9#9: *1 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 192.168.65.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:3000", referrer: "http://localhost:3000/"
> spa-management-vue    | 192.168.65.1 - - [10/Nov/2023:14:42:32 +0000] "GET /favicon.ico HTTP/1.1" 500 177 "http://localhost:3000/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0" "-"
> ```
> 
> @odzhychko have you encountered this as well?

It works for me. I do not get this error.

**Comment by mhuster23** [comment]
@odzhychko @nkoester nvm, the issue was my local setup

**Pull request merged**